baremetalpxeserver/baremetalpxeserver/pxeserveragent.py

```
@@ -1181,12 +1343,20 @@ def download_imagestore(self, req):
         # SUSE
         ret5 = bash_r("cp %s %s && chmod 777 %s" % (os.path.join(mount_path, "boot/*/loader/linux"), vmlinuz_file_path, vmlinuz_file_path))
         ret6 = bash_r("cp %s %s && chmod 777 %s" % (os.path.join(mount_path, "boot/*/loader/initrd"), initrd_file_path, initrd_file_path))
-        # ky10
+        # ns10
         ret7 = bash_r("cp %s %s && chmod 777 %s" % (os.path.join(mount_path, "images/pxeboot/vmlinuz"), vmlinuz_file_path, vmlinuz_file_path))
```

Ruff (0.11.9)

`bash_r` may be undefined, or defined from star imports

(F405)